python/ray/experimental/state.py

```diff
@@ -16,6 +17,10 @@
 from ray.utils import (decode, binary_to_object_id, binary_to_hex,
                        hex_to_binary)
 
+# These channels must be kept up-to-date with the C++ codebase
+LOCAL_SCHEDULER_INFO_CHANNEL = b"local_schedulers"
```

Comment:
This is in `gcs_utils.py`.

Reply:
I couldn't find this in `gcs_utils.py`. Should I add it? It is in `monitor.py` as well.